testkit/src/main/java/org/apache/calcite/test/SqlOperatorTest.java

```diff
@@ -5167,6 +5167,18 @@ private static void checkIf(SqlOperatorFixture f) {
     f.checkNull("array_concat(cast(null as integer array), array[1])");
   }
 
+  /** Tests {@code ARRAY_DISTINCT} function from Spark. */
```

Comment:
```
  /**
   * Test case for
   * <a href="https://issues.apache.org/jira/browse/CALCITE-5657"> [CALCITE-5657]
   * Add ARRAY_DISTINCT function (enabled in Spark library)
   * </a>.
   */
```
Maybe better.

Reply:
may not. you can refer other functions like array_remove in the calcite code